docs/lib/graphqlapi/fragments/flutter/getting-started/30_initapi.md

```diff
@@ -0,0 +1,24 @@
+To initialize the Amplify API category you call `Amplify.addPlugin()` method for each category. To complete initialization call `Amplify.configure()`.
```

Comment:
This contradicts below where you instantiate an instance. Did Praveen's change for this get merged yet?

Reply:
Not too sure how this contradicts?  We use similar phrasing for all of our other categories: 

To initialize the Amplify Auth and Storage categories you call `Amplify.addPlugin()` method for each category. To complete initialization call `Amplify.configure()`.

I don't think Praveen's changes have been merged yet.  Once they are, we will need to update for all categories appropriately though.

Reply:
Thanks you're right, I have updated our API docs to be consistent with Praveen's latest changes.